### DISCUSSION TITLE
[월드 1.1.7] 멀티 테스트 onLeave호출 문제 (index.ts)

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#611] [5. Q&A]
월드 1.1.7 입니다 

플레이어가 떠나도 서버에서 onLeave 호출이 안되는것 같습니다...

https://user-images.githubusercontent.com/50188795/163089667-89c466a1-798d-48fd-a8d9-6481ddd41354.mp4

![image](https://user-images.githubusercontent.com/50188795/163089781-5bbd67fb-9dfd-4d55-986d-be57b9541a02.png)

https://user-images.githubusercontent.com/50188795/163105291-40167cdd-1d44-4ef2-9a89-520189003b1d.mp4

**Comment by Deytooh** [comment]
네 저도 똑같은 문제 있어요.... 유니티에서 OnLeave 안돼요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 현상을 에디터 -모바일간 멀티플레이로 테스트를 진행 했습니다.
에디터 유저가 방을 나갈 경우 OnLeave 호출이 안되지만,
모바일 유저가 방을 나갈 경우에는 정상적으로 호출이 되는데요
@smilejsu82 @Deytooh 님도 모바일 유저가 방을 나갈때는 잘 되고 있는지 한번 확인 부탁드립니다.

**Reply by Deytooh** [reply]
네 모바일에서 잘 되고있는데 유니티에서 안되는거 확인했습니다 ! 근데 테스트 할때 유니티에서 onleave 안돼는거 불편할 수도 있어요 ㅠㅠ

**Reply by Zepeto-TechEvangelist** [reply]
네, 동일한 현상이 재현되었으므로 원인 파악 해 보겠습니다.

**Reply by smilejsu82** [reply]
> 네, 동일한 현상이 재현되었으므로 원인 파악 해 보겠습니다.

모바일과 애뮬레이터 확인 했습니다 잘 동작 하네요 
![image](https://user-images.githubusercontent.com/50188795/163543881-b804a21a-4b67-4e32-ab21-2bd1287b3475.png)

에디터에서 안되는건 많이 불편 할거 같아요
